src/jdk.crypto.cryptoki/share/classes/sun/security/pkcs11/Config.java

```diff
@@ -62,27 +60,16 @@ final class Config {
     private static final String osArch;
 
     static {
-        @SuppressWarnings("removal")
-        List<String> props = AccessController.doPrivileged(
-            new PrivilegedAction<>() {
-                @Override
-                public List<String> run() {
-                    return List.of(
-                        System.getProperty(
-                            "sun.security.pkcs11.allowSingleThreadedModules",
-                            "true"),
-                        System.getProperty("os.name"),
-                        System.getProperty("os.arch"));
-                }
-            }
-        );
-        if ("false".equalsIgnoreCase(props.get(0))) {
+        String allowSingleThreadedModules =
+            System.getProperty(
+                "sun.security.pkcs11.allowSingleThreadedModules", "true");
+        if ("false".equalsIgnoreCase(allowSingleThreadedModules)) {
             staticAllowSingleThreadedModules = false;
         } else {
             staticAllowSingleThreadedModules = true;
         }
-        osName = props.get(1);
-        osArch = props.get(2);
+        osName = System.getProperty("os.name");
+        osArch = System.getProperty("os.arch");
```

Comment:
osName and osArch seem to be unused; they could be removed but might be out of scope for this PR.

Reply:
Good spot; I removed them - I'm ok with making small changes as long as they are within the scope of the code being modified.